study.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const self = require("sdk/self");
+const { when: unload } = require("sdk/system/unload");
+const tabs = require("sdk/tabs");
+
+const shield = require("./lib/shield/index");
+
+const studyConfig = {
+  name: self.addonId,
+  days: 28,
+  surveyUrls: {
```

Comment:
this will do no surveys anywhere, as written.  If you want them, this needs fill.